### PR TITLE
Upgrade to SKY UX 4

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux?_sp.spa=pr-skyux4](https://developer.blackbaud.com/skyux?_sp.spa=pr-skyux4)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux?_sp.spa=pr-skyux4](https://developer.blackbaud.com/skyux?_sp.spa=pr-skyux4)

**Comment by Blackbaud-DeniseCisneros** [comment]
Code and code block components are coming up as 404s. I can't remember if we decided to pull them completely from the component list or just add links to the Stache docs.

File: package.json [hunk 1]
```diff
@@ -9,88 +9,80 @@
...
-    "@skyux-sdk/e2e": "3.2.0",
-    "@skyux-sdk/pact": "3.3.1",
-    "@skyux-sdk/testing": "3.3.0",
-    "@types/core-js": "2.5.3",
-    "codelyzer": "4.5.0",
+    "@skyux-sdk/builder": "4.5.1",
+    "@skyux-sdk/e2e": "4.0.0",
+    "@skyux-sdk/testing": "4.2.2",
+    "codelyzer": "5.2.2",
     "core-js": "2.6.11",
```
Comment (Blackbaud-SteveBrush) [inline package.json:82]: `core-js` is handled by Builder, so this should be removed.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux?_sp.spa=pr-skyux4](https://developer.blackbaud.com/skyux?_sp.spa=pr-skyux4)

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Pull request merged**